source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
```

Comment:
I am not really sure to understand what you call "unofficial" and why. The mattermost package is the "official" AUR package. 

> Please append (Unofficial) to the end of the heading?
What do you mean exactly?

> Also, since this is unoffical, please move this section below the heading 
Do you mean the section shall be structured this way:
Archlinux
Build and install manually
Build and install with AUR